nginx-config.yml.sample

```diff
@@ -5,6 +5,7 @@ instances:
       command: metrics
       arguments:
           status_url: http://127.0.0.1/status
+          remote_monitoring: false #new users should leave remote_monitoring = true
```

Comment:
Maybe I would be more verbose. I was thinking on something like:
```
        # New users should leave this property as `true`, to identify the
        # monitored entities as `remote`. Setting this property to `false` (the
        # default value) is deprecated and will be removed soon, disallowing
        # entities that are identified as `local`.
        # Please check the documentation to get more information about local
        # versus remote entities:
        # https://github.com/newrelic/infra-integrations-sdk/blob/master/docs/entity-definition.md
```
But not sure. WDYT? 

CC: @ikavanaghNR

Reply:
👍 I changed my comments as well.